server/metrics/metrics.go

```diff
@@ -27,6 +27,10 @@ import (
 	promreporter "github.com/uber-go/tally/prometheus"
 )
 
+const (
+	DefaultReportedNameSpace string = "default_namespace"
```

Comment:
can you move this in types.go inside the server/types package and let's call it `DefaultNamespace`

Reply:
This is done.